src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -1201,8 +1200,8 @@ private DetectProperties() {
             .setGroups(DetectGroup.PIP, DetectGroup.GLOBAL)
             .build();
 
-    public static final NoneEnumListProperty<PnpmDependencyTypeV2> DETECT_PNPM_DEPENDENCY_TYPES_EXCLUDED =
-        NoneEnumListProperty.newBuilder("detect.pnpm.dependency.types.excluded", NoneEnum.NONE, PnpmDependencyTypeV2.class)
+    public static final NoneEnumListProperty<PnpmDependencyType> DETECT_PNPM_DEPENDENCY_TYPES_EXCLUDED =
+        NoneEnumListProperty.newBuilder("detect.pnpm.dependency.types.excluded", NoneEnum.NONE, PnpmDependencyType.class)
             .setInfo("pnpm Dependency Types", DetectPropertyFromVersion.VERSION_7_11_0)
             .setHelp(createDefaultDrivenPropertyHelpText("Pnpm dependency type", "detect.pnpm.dependency.types"))
```

Comment:
I think if you merge `master` into this branch, the `createDefaultDrivenPropertyHelpText` method has changed.
There should be comments explaining why you should drop the `"detect.pnpm.dependency.types"` parameter.